### ISSUE TITLE
Issues when importing psi4

### ISSUE THREAD

**Issue by jnotse** [psi4/psi4#2329]
Hello, i recently installed the psi4 package using the conda prompt (Im using miniconda) with this code:
`conda create -n p4env psi4 python=3.8 -c psi4 -c conda-forge`
I configurated the paths, adding the ones given by `psi4 --psiapi`, but when importing the package in Jupyter Notebook I still have issues. It seems like the `import psi4` works, but when pressing tab after `psi4.` there are no suggestions, but with `psi4._` I get only seven suggestions (`__doc__`, `__file__`,`__loader__`, ...). So, when trying `psi4.set_memory(int(5e8))`, I get `AttributeError: module 'psi4' has no attribute 'set_memory'`. Thanks for any suggestions (Im new to psi4 and just recently started using Python, and GitHub).

**Comment by loriab** [comment]
Thanks for trying out psi4! If you're on windows, your conda install line is just right. If you're on linux or mac, you don't actually need the `-c conda-forge`, though it's harmless. Since you're installing from conda, the `psi4 --psiapi` is unnecessary --- activating the environment suffices.

I don't have much personal experience with tab help in jupyter, but `help(psi4.set_memory)` returns a sensible docstring for me in interactive python. What you describe does sound with `AttributeError` does sound wrong. If the in-jupyter help is giving you trouble, searching the generated API (https://psicode.org/psi4manual/master/psi4api.html) may work better. If you're new, perhaps check out the tutorials at https://psicode.org/psi4manual/master/index_tutorials.html .

**Comment by jnotse** [comment]
> 
> 
> are you on Windows? make sure you're getting the v1.4.1, not v1.4

Yes, Im using Windows. I checked with `conda list -n p4env` and it is v1.4.1

**Comment by Astlaan** [comment]
Any update on this? I tried installing Psi4 on Windows via conda. I tried several python versions (3.10, 3.11, 3.12). It always failed and I got the DLL error, even when simply running the `psi4` command.

Installing via binary installer did work.

**Comment by loriab** [comment]
Is it that original error `ImportError: DLL load failed while importing core`? For Windows psi4, there's also a linalg error floating around (solve by using `psi4=1.9.1=*_8` or `_10` builds), a "can't load module" (approx words) error (solve by constraining `libgcc<14.3`, and a certain-tests-failing error when installed in the presence of `rdkit` (error comes and goes with identical environments). As of late last week the psi4 windows conda package was provably working, though. These are just a few facts to orient what trouble you may be hitting.

**Comment by Astlaan** [comment]
> Is it that original error `ImportError: DLL load failed while importing core`? For Windows psi4, there's also a linalg error floating around (solve by using `psi4=1.9.1=*_8` or `_10` builds), a "can't load module" (approx words) error (solve by constraining `libgcc<14.3`, and a certain-tests-failing error when installed in the presence of `rdkit` (error comes and goes with identical environments). As of late last week the psi4 windows conda package was provably working, though. These are just a few facts to orient what trouble you may be hitting.

@loriab Yes, the error that I got was `ImportError: DLL load failed while importing core`.
I think I remember reading somewhere that one the the things that could cause this is if the CPU doesn't support AVX512 instructions since the psi4 conda package didn't include non-AVX512 instructions (don't know if this is correct or not).  My CPU indeed only supports AVX and AVX2, not AVX512.

**Comment by Astlaan** [comment]
@TiborGY using the CMD (both env creation + installation, and execution of psi4) still gives me the same error.

@loriab 
Using ` conda create -n p4dev2 psi4 libgcc<14.3 python=3.12 -c conda-forge; conda activate p4dev2`

appears to work, I get this:

```powershell
(p4dev2) PS C:\Users\user> psi4
Traceback (most recent call last):
  File "C:\Users\user\anaconda3\envs\p4dev2\Scripts\psi4-script.py", line 265, in <module>
    raise KeyError("The file %s does not exist." % args["input"])
KeyError: 'The file input.dat does not exist.'
```

maybe then it's just a matter of updating the conda recipe?

**Comment by Astlaan** [comment]
Opened a PR to fix the recipe:
https://github.com/conda-forge/psi4-feedstock/pull/42
